packages/chat/src/contexts/RuntimeContext/audio-controller.ts

```diff
@@ -0,0 +1,40 @@
+export class AudioController {
```

Comment:
This (and a few other files related to audio) were added recently to the 'old' chat, so i copied them to this chat here, so we have the same functionality